nyc_taxis/TEST_PROCEDURES.md

```diff
@@ -0,0 +1,55 @@
+# Test Procedures
+
+## searchable_snapshot 
+
+A test procedure for measuring performance for Searchable Snapshots feature. It runs the same search query as the default test procedure `append-no-conflicts`.
+
+In contrast with `append-no-conflicts` which running query on index with local storage, this test procedure runs query on index backed by a remote snapshot.
+
+The test procedure will create a remote snapshot that stored in Amazon S3, so an Amazon S3 bucket for storing the snapshot and an AWS account user credential that has permission to access the bucket are required to run the test procedure. To learn more about configuring Amazon S3 as a snapshot repository, see the [OpenSearch docs](https://opensearch.org/docs/latest/opensearch/snapshots/snapshot-restore#amazon-s3).
+
+Searchable Snapshots feature is supported by OpenSearch since version 2.4.0, see the [OpenSearch docs](https://opensearch.org/docs/2.4/opensearch/snapshots/searchable_snapshot) to learn more.
+
+### Parameters
+
+#### The test procedure allows to overwrite the following parameters using `--workload-params`:
+
+Additional parameters other than those mentioned in [README](README.md) of NYC taxis workload:
```

Comment:
allows the following parameters to be specified using --workload-params in addition to those mentioned in ...